### PR TITLE
Support importing any custom model package

### LINKED ISSUE #2162: when contribute model test, get error: NameError: name '***' is not defined
I‘m testing distribute model to sqlflow in the docker on my mac, but I have some problem as follows.

image

but i get the following error

image

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/sql-machine-learning/sqlflow/issues/2162

### CODE REVIEW

File: pkg/sql/codegen/tensorflow/template_train.go [hunk 1]
```diff
@@ -112,6 +113,12 @@ feature_columns = eval(feature_columns_code)
 train_max_steps = {{index .TrainParams "max_steps" | attrToPythonValue}}
 train_max_steps = None if train_max_steps == 0 else train_max_steps
 
+model_import_name = sqlflow_submitter.get_import_name("""{{.Estimator}}""")
```
Comment (Yancey0623) [inline pkg/sql/codegen/tensorflow/template_train.go]: Move to Python code can keep the template cleaner?

File: pkg/sql/pai_submitter.go [hunk 2]
```diff
@@ -559,6 +561,25 @@ func achieveResource(cwd, entryCode, requirements, tarball string) error {
 		return fmt.Errorf("failed %s, %v", cmd, err)
 	}
 
+	// add any other custom model packages
+	fmt.Printf("submitting with model dep: %s\n", estimator)
```
Comment (wangkuiyi) [inline pkg/sql/pai_submitter.go]: dep => definition

File: pkg/sql/codegen/pai/template_tf.go [hunk 3]
```diff
@@ -164,8 +164,14 @@ if is_estimator:
 else:
     model.load_file("{{.OSSModelDir}}", "model_save")
 
+model_import_name = sqlflow_submitter.get_import_name("""{{.Estimator}}""")
+try:
+    globals()[model_import_name] = __import__(model_import_name)
+except Exception as e:
+    print("failed to import %s: %s" % (model_import_name, e))
```
Comment (wangkuiyi) [inline pkg/sql/codegen/pai/template_tf.go]: This segment of code appears 5 times in this PR.  It seems should be defined as a function

```python
def sqlflow_submitter.import_model_def(estimator_cls_name):
    model_import_name = sqlflow_submitter.get_import_name(estimator_cls_name)
    try:
        globals()[model_import_name] = __import__(model_import_name)
    except Exception as e:
        print("failed to import %s: %s" % (model_import_name, e))
```

and we call it here as

```python
sqlflow_submitter.import_model_def("""{{.Estimator}}""")
```

**Review by wangkuiyi — APPROVED** [review]

**Review by wangkuiyi — CHANGES_REQUESTED** [review]
@typhoonzero This PR includes changes to .travis.yml, which reverted some of my recent changes. How about revert these changes to .travis.yml?

**Review by wangkuiyi — APPROVED** [review]

**Pull request merged**